<launch> 
 <include file="$(find pr2_machine)/$(env ROBOT).machine" />

	<rosparam ns="/robot_description_planning" command="load" file="$(find pr2_arm_navigation_config)/config/planning_groups.yaml" />
	<node pkg="stomp_motion_planner" name="stomp_motion_planner" type="stomp_motion_planner" respawn="false" output="screen" clear_params="true">
                <rosparam command="load" file="$(find stomp_motion_planner)/config/pr2_both_arms_stomp_config.yaml" />
		<remap from="collision_map" to="collision_map_occ"/>

		<rosparam command="load" file="$(find stomp_motion_planner)/config/params.yaml" />
		<rosparam command="load" file="$(find pr2_arm_navigation_config)/config/robot_padding.yaml" />
                <param name="use_chomp" value="true"/>

                <param name="learning_rate" value="0.05" />
                <param name="max_iterations" value="500"/>
                <param name="max_iterations_after_collision_free" value="100" />
                <param name="smoothness_cost_weight" value="0.00000001"/>
                <param name="obstacle_cost_weight" value="0.5" />
                <param name="joint_update_limit" value="0.01" />
                <param name="ridge_factor" value="0.0" />
                <param name="use_pseudo_inverse" value="false" />
                <param name="pseudo_inverse_ridge_factor" value="0.0001" />

                <param name="add_randomness" value="false" />
                <param name="use_hamiltonian_monte_carlo" value="false" />
                <param name="hmc_discretization" value="0.02" />
                <param name="hmc_stochasticity" value="0.001" />
                <param name="hmc_annealing_factor" value="0.95" />


	</node>
</launch>
